cohort-engine/src/main/java/com/ibm/cohort/engine/measure/FHIRClientContext.java

```diff
@@ -6,13 +6,13 @@
 
 package com.ibm.cohort.engine.measure;
 
-import ca.uhn.fhir.context.FhirContext;
-import ca.uhn.fhir.rest.client.api.IGenericClient;
 import com.ibm.cohort.fhir.client.config.DefaultFhirClientBuilderFactory;
 import com.ibm.cohort.fhir.client.config.FhirClientBuilder;
 import com.ibm.cohort.fhir.client.config.FhirClientBuilderFactory;
 import com.ibm.cohort.fhir.client.config.FhirServerConfig;
 
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+
 /**
  * A wrapper around the different HAPI client connections required by the engine.
  *
```

Comment:
You've done a good job updating the comments in other sections of the API, but I think I would like to also add some detail here that states... "FHIRClientContext objects are relatively expensive to create and should be reused as much as possible."